### ISSUE TITLE
Redesign application header to fit to blog

### ISSUE THREAD

**Issue by lennerd** [sozialhelden/wheelmap#403]
I started to redesign the application header based on the layout of the new blog theme. Here are some results of the first iteration.

![desktop-accessibility-dropdown](https://cloud.githubusercontent.com/assets/1609692/18639377/f66ca854-7e93-11e6-9a92-562a9e9d5401.png)
*The classical dropdown. No feature changes here.*

![desktop-category-dropdown](https://cloud.githubusercontent.com/assets/1609692/18639378/f79cdcbc-7e93-11e6-8557-da17d3bbb0e7.png)
*Same for the category dropdown. I'm not sure, if the "Toggle all" button at the bottom is seen as a button at all. The label should add more clarity about what the button does though.*

![mobile](https://cloud.githubusercontent.com/assets/1609692/18639380/f9535054-7e93-11e6-986b-d528019fa7ea.png)
*Again no feature changes here.*

![mobile-search](https://cloud.githubusercontent.com/assets/1609692/18639382/fa9725bc-7e93-11e6-9a7c-25e86dcc6bf9.png)
*I would like to move out the other buttons, when the user clicks on the search button. This way he or she can focus on the input and we have smoother transition between both states.*

![mobile-accessibility](https://cloud.githubusercontent.com/assets/1609692/18639391/fbda6d62-7e93-11e6-960f-f3c0e04637cd.png)
*This is the biggest change I think. I never was a big fan of the popup info icons inside the dropdown. Certainly not optimized for mobile use. Instead of the dropdown, I would like to remove all arrow icons from the buttons and open a simple overlay with both the selectable filter and its description.*

I'm not sure about how to develop this new layout beside the old and legacy layout without destroying everything. Feedback is welcome!

I will use this issue to report next iteration and as reference for the corresponding PR.

**Comment by holgerd** [comment]
@lennerd It looks great imho but also like a lot of work. Any chance you can finish this this month? If not, can you break it down and finish a little bit of this this month?

**Comment by Hoverbear** [comment]
I agree it looks very good. :) Only thing I noticed was on the news site the logo is a bit smaller.

**Comment by lennerd** [comment]
Thanks for the feedback. I'm able to come back to this on Friday.

@holgerd Good point I also had in mind all the time. Reason why I did so much changes was that I didn't want to design a half backed layout but see what is possible and how it would fit together. And I also fixed some things on the way I was annoyed by for a longer time now. These changes are not necessary for a first version (e.g. the accessibility overlay on mobile devices). 

@Hoverbear Thanks! I will check that. Plan is to have some bigger similarities to see both sites belong together. It's not about have the exact same header, so I think some smaller changes to make it more suitable for the web application are possible.
